docs/user/customization.md

```diff
@@ -70,6 +70,25 @@ push down, editor.action.moveLinesDownAction
 
 Now when you say eg "push down air and bat", cursorless will first select the two tokens with a gray hat over the `a` and `b`, then issue the VSCode command `editor.action.moveLinesDownAction`, and then restore your original selection.
 
+## \[Experimental\] Cursorless custom regex scopes
+
+You can define custom scopes in `user/cursorless-settings/experimental/regex_scope_types.csv`
+
+> [!TIP]
+> Use `visualize [scope]` to see a live preview of the regex scope (must save `regex_scope_types.csv` to see changes)
+
+---
+Command `visualize dotted` with the following scope type `dotted,"[\w.]+"`
+
+![image](https://github.com/cursorless-dev/cursorless/assets/13824365/b54263a3-74b7-40d5-a1aa-cd7bbe86430b)
+
+You can then use commands such as
+* `take dotted sun` to select "user.text" (line 10)
+* `take dotted urge` to select "user.chrome_mod" (line 11)
+
+> [!NOTE]
+> Examples are at https://github.com/cursorless-dev/cursorless/wiki/Custom-regex-scopes
+
```

Comment:
I might add a warning admonition here about the fact that the regexes can't cross line boundaries. We could also point out that the regexes have the unicode flag set? Idk. Fwiw [here](https://github.com/cursorless-dev/cursorless/blob/02e70878d3115cabd85ed23b11adb4de6868e7d0/packages/cursorless-engine/src/processTargets/modifiers/scopeHandlers/RegexScopeHandler.ts#L60)'s where we ingest their regexes

Reply:
Added warning and info link to implementation.

Reply:
Took many tries, but figured out how it's doing relative links. I mean the result seems intuitive, but not what the GitHub main docs made me think should be done.